android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/FormAnswer.kt

```diff
@@ -0,0 +1,78 @@
+package co.nimblehq.ic.kmm.suv.android.ui.components
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldColors
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.ic.kmm.suv.android.ui.theme.AppTheme
+import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+
+typealias Placeholder = String
+
+@Composable
+fun FormAnswer(
+    placeholders: List<Placeholder>,
+    onTextChange: (Pair<Int, String>) -> Unit,
```

Comment:
`Pair<Int, String>` is too value, we need a DTO

Reply:
Please ignore this, I've changed on all answers for consistency: https://github.com/suho/kmm-ic/pull/123/files#diff-0a5f71ca5c346d1a7908daa87906e87bd8d038716044a0a7b5526578016255e1R34